includes/GlobalFunctions.php

```diff
@@ -3817,6 +3817,21 @@ function wfWaitForSlaves( $wiki = false ) {
 		$dbw = $lb->getConnection( DB_MASTER, array(), $wiki );
 		$pos = $dbw->getMasterPos();
 		$lb->waitForAll( $pos, $wiki );
+
+		// Wikia change - begin
+		// OPS-6313 - sleep-based implementaion for cluster G (it uses consul-powered DB config)
+		if ( $lb->parentInfo()['id'] === 'main-c7' ) {
```

Comment:
could you check if dbhost youre connecting to is in .consul domain or something generic like that? hardcoding cluster name is not the best option IMHO